partner_communication_revision/data/demo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="template" model="mail.template">
+            <field name="name">Donation - Thank You Letter</field>
+            <field name="model_id" ref="partner_communication.model_partner_communication_job"/>
+            <field name="use_default_to" eval="True"/>
+            <field name="subject">Donation Receipt</field>
+            <field name="body_html" type="html">
+                <div>
+                    % set invoice_lines = object.get_objects()
+                </div>
+                <div>
+                    ${partner.salutation},
+                    <br/>
+                    <br/>
+                    We thank you for the donation of CHF ${donations[0]}.-
+                    % if donations[1]:
```

Comment:
`donations` will fail at template rendering since the variable is not declared. You can remove this part for simplicity.

Reply:
I indeed gathered random parts of templates without checking that the whole thing was valid :( Should be better now.